configs/frontend_config/config.json

```diff
@@ -1 +1,926 @@
-{}
\ No newline at end of file
+{
+  "@openmrs/esm-patient-chart-app": {
+    "extensionSlots": {
+      "patient-chart-dashboard-slot": {
+        "add": [
+          "nav-group#HIVCareTreatment"
+        ],
+        "configure": {
+          "nav-group#HIVCareTreatment": {
+            "title": "HIV Care and Treatment",
+            "slotName": "hiv-care-and-treatment-group-slot",
+            "isExpanded": false
+          }
+        }
+      },
+      "hiv-care-and-treatment-group-slot": {
+        "add": [
+          "hiv-patient-summary-slot",
+          "program-management-slot",
+          "clinical-visits-slot",
+          "general-counselling-slot",
+          "partner-notification-slot"
+        ]
+      },
+      "hiv-patient-summary-slot": {
+        "add": [
+          "encounter-tiles-group"
+        ],
+        "configure": {
+          "encounter-tiles-group": {
+            "tilesDefinitions": [
+              {
+                "columns": [
+                  {
+                    "id": "artCohort",
+                    "title": "ART Cohort",
```

Comment:
Something we definitely need to work out is how we support i18n for this. Otherwise, this package becomes useless outside of an English-speaking context, which would be unfortunate.

My suggestion would be that instead of hard-coding `title`, `label`, etc. properties, we provide translation keys, e.g., `artCohort` and then in the `configure` section, we can provide the `Translation overrides` with the default values for those keys. That should work with only minor code changes _and_ enable us to make this content package localizable.